packages/core/components/DataTable.jsx

```diff
@@ -1,759 +0,0 @@
-      break
-    case 'Line' || 'Bar' || 'Combo' || 'Pie' || 'Deviation Bar' || 'Paired Bar':
-      if (!runtimeData) return <Loading />
-      break
-    default:
-      if (!runtimeData) return <Loading />
-      break
-  }
-
-  const rawRows = Object.keys(runtimeData)
-  const rows = isVertical
-    ? rawRows.sort((a, b) => {
-        let sortVal = 0
-        if (config.type === 'map' && config.columns) {
-          sortVal = customSort(runtimeData[a][config.columns[sortBy.column].name], runtimeData[b][config.columns[sortBy.column].name])
-        }
-        if (config.type === 'chart' || config.type === 'dashboard') {
-          sortVal = customSort(runtimeData[a][sortBy.column], runtimeData[b][sortBy.column])
-        }
-        return sortVal
-      })
-    : rawRows
-
-  const genMapRows = rows => {
-    const allrows = rows.map(row => {
-      return (
-        <tr role='row'>
-          {Object.keys(columns)
-            .filter(column => columns[column].dataTable === true && columns[column].name)
-            .map(column => {
-              let cellValue
-
-              if (column === 'geo') {
-                const rowObj = runtimeData[row]
-                const legendColor = applyLegendToRow(rowObj)
-
-                var labelValue
-                if ((config.general.geoType !== 'single-state' && config.general.geoType !== 'us-county') || config.general.type === 'us-geocode') {
-                  labelValue = displayGeoName(row)
-                } else {
-                  labelValue = formatLegendLocation(row)
-                }
-
-                labelValue = getCellAnchor(labelValue, rowObj)
-                cellValue = (
-                  <>
-                    <LegendCircle fill={legendColor[0]} />
-                    {labelValue}
-                  </>
-                )
-              } else {
-                // check for special classes
-                let specialValFound = ''
-                if (config.legend.specialClasses && config.legend.specialClasses.length && typeof config.legend.specialClasses[0] === 'object') {
-                  for (let i = 0; i < config.legend.specialClasses.length; i++) {
-                    if (config.legend.specialClasses[i].key === config.columns[column].name) {
-                      if (String(runtimeData[row][config.legend.specialClasses[i].key]) === config.legend.specialClasses[i].value) {
-                        specialValFound = config.legend.specialClasses[i].label
-                      }
-                    }
-                  }
-                }
-                cellValue = specialValFound ? displayDataAsText(specialValFound, column) : displayDataAsText(runtimeData[row][config.columns[column].name], column)
-              }
-
-              return (
-                <td tabIndex='0' role='gridcell' onClick={e => (config.general.type === 'bubble' && config.general.allowMapZoom && config.general.geoType === 'world' ? setFilteredCountryCode(row) : true)}>
```

Comment:
pretty sure and onClick on a <td> isn't 508 compliant. can we handle this some other way? @adamdoe said the feature is broken anyway.

@mpallansch @Atash3000